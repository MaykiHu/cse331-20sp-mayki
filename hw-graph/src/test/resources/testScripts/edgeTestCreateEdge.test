#Create a node
CreateNode 1

#Create another node
CreateNode 2

#Create an edge
CreateEdge n1 n2 e1