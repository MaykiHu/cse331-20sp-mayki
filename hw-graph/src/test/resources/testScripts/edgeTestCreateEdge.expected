#Create a node
created node n1

#Create another node
created node n2

#Create an edge
created edge n1 n2 e1